### ISSUE TITLE
multi-threading is not working

### ISSUE THREAD

**Issue by lucacozzuto** [Psy-Fer/deeplexicon#19]
Dear authors,
I used multi-threading and I got only a few reads generated. While removing that option gives me no error. Also using -t 1 is generating an error.

Thanks,
Luca

**Comment by Psy-Fer** [comment]
Hello,

Could you please give me the full command you used, and with which version of the software, as well as any errors you are encountering.

Thanks,
James

ps @lpryszcz  could you please keep an eye on this one :)

**Comment by lucacozzuto** [comment]
I'm using the image from @lpryszcz actually. The command line is the following
```
deeplexicon_sub.py dmux -f multi -m resnet20-final.h5 -t 1 -p ./ > pU---1_demux.tsv

```

**Comment by Psy-Fer** [comment]
Ahh, deeplexicon_sub.py doesn't have a -t for threads, but for a --test command.

please use deeplexicon_multi.py with `--threads` argument

**Comment by lucacozzuto** [comment]
Aha! Can I suggest changing this as well? This is a bit misleading :) Sorry for this but it took me a day to figure it out the problem :)

**Comment by lpryszcz** [comment]
I'm sorry Luca. I kept old version for back-compatibility (I guess older versions are still needed for training of the model). Everything is described in the README https://github.com/Psy-Fer/deeplexicon#source-and-install-requirements-gpu-experimental
@Psy-Fer could you please update the documentation accordingly (https://psy-fer.github.io/deeplexicon/)?

**Comment by Psy-Fer** [comment]
Yep,. I'm on break at the moment and will be back in the new year. 
We can probably remove the test flag too so we can free up the -t argument for threads
